draft-ietf-ppm-dap.md

```diff
@@ -789,26 +791,21 @@ include any information specifying a particular batch (see {{query}}). The
 Leader selects a recent batch to aggregate. The Leader MUST select a batch that
 has not yet been associated with a collection job.
 
-In addition to the minimum batch size common to all query types, the
-configuration may include a parameter `max_batch_size` that determines maximum
-number of reports per batch.
-
-If the configuration does not include `max_batch_size`, then the Aggregators
-can output any batch size that is larger than or equal to `min_batch_size`.
-This is useful for applications that are not concerned with sample size, i.e.,
-the privacy guarantee is not affected by the sampling rate of the population,
-therefore a larger than expected batch size does not weaken the designed privacy
-guarantee.
+The Aggregators can output batches of any size that is larger than or equal to
+`min_batch_size`. This is useful for applications that are not concerned with
+sample size, i.e., the privacy guarantee is not affected by the sampling rate of
+the population, therefore a larger than expected batch size does not weaken the
+designed privacy guarantee.
 
-Implementation note: The goal for the Aggregators is to aggregate the same
-number of reports in each batch. The target batch size is deployment-specific,
+Implementation note: The target batch size, if any, is implementation-specific,
 and may be equal to or greater than the minimum batch size. Deciding how soon
-batches should be output is also deployment-specific. Exactly sizing batches may
-be challenging for Leader deployments in which multiple, independent nodes
+batches should be output is also implementation-specific. Exactly sizing batches
+may be challenging for Leader deployments in which multiple, independent nodes
```

Comment:
I don't object, but why change "deployment" to "implementation" here?

Reply:
Saying something is "implementation-specific" is (in my experience) much more common than "deployment-specific". Prior to this document, I am not sure I have seen the use of "deployment" in this fashion; instead, typically "implementation" would be used.

Reply:
Thinking about this a little more, at least in my head:

* An "implementation" is a specific software implementation of some DAP protocol actor. Examples would include Janus or Daphne.
* A "deployment" is a practical deployment of DAP; it would use one or more implementations of the DAP protocol actors, as well as defining tasks (or deciding how tasks can be dynamically configured, e.g. taskprov or some other task-provisioning scheme), defining other configuration parameters, deciding how the software implementations are run on machines, and everything else that comes with deploying software. Examples would include Divvi Up.